modules/programs/taskwarrior.nix

```diff
@@ -0,0 +1,106 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.taskwarrior;
+
+  includeTheme = location:
+    if (location == null) then ""
```

Comment:
Can remove the parentheses in each if-condition in the file. For example, `if location == null then …` and `else if isString location then …`